features/frameworks/mbed-trace/mbed-trace/mbed_trace.h

```diff
@@ -59,22 +59,24 @@ extern "C" {
 #include <stdarg.h>
 
 #ifndef YOTTA_CFG_MBED_TRACE
-#define YOTTA_CFG_MBED_TRACE 0
+#define YOTTA_CFG_MBED_TRACE            0
 #endif
 
 #ifndef YOTTA_CFG_MBED_TRACE_FEA_IPV6
-#define YOTTA_CFG_MBED_TRACE_FEA_IPV6 1
+#define YOTTA_CFG_MBED_TRACE_FEA_IPV6   1
 #else
 #warning YOTTA_CFG_MBED_TRACE_FEA_IPV6 is deprecated and will be removed in the future! Use MBED_CONF_MBED_TRACE_FEA_IPV6 instead.
 #define MBED_CONF_MBED_TRACE_FEA_IPV6 YOTTA_CFG_MBED_TRACE_FEA_IPV6
 #endif
 
 #ifndef MBED_CONF_MBED_TRACE_ENABLE
-#define MBED_CONF_MBED_TRACE_ENABLE 0
+#define MBED_CONF_MBED_TRACE_ENABLE     0
```

Comment:
If required default trace library can be kept as enabled, but config option should not be overwritten.
```suggestion
#define MBED_CONF_MBED_TRACE_ENABLE     1
```

If required default trace library can be kept as enabled, but config option should not be overwritten.